pkgs/tools/misc/youtube-dl/default.nix

```diff
@@ -1,4 +1,5 @@
-{ stdenv, fetchurl, makeWrapper, python, zip, ffmpeg, pandoc ? null }:
+{ stdenv, fetchurl, makeWrapper, pythonPackages, zip, ffmpeg
```

Comment:
Passing the entire `pythonPackages` package set instead of just the required attributes obscures the dependencies of the expression. That doesn't seem like an improvement over the previous state.